cmd/clusterctl/README.md

```diff
@@ -41,10 +41,12 @@ https://github.com/kubernetes-sigs/cluster-api/issues/158 and https://github.com
 1. Create a cluster:
 
    ```shell
-   ./clusterctl create cluster --provider <provider> -c cluster.yaml -m machines.yaml -p provider-components.yaml -a addons.yaml
+   ./clusterctl create cluster --provider <provider> --bootstrap-type <bootstrap-type> -c cluster.yaml -m machines.yaml -p provider-components.yaml -a addons.yaml
    ```
 
-To choose a specific minikube driver, please use the `--vm-driver` command line parameter. For example to use the kvm2 driver with clusterctl you would add `--vm-driver kvm2`
+Currently two `bootstrap-type` options are supported - `kind` and `minikube`.
+
+If you are using minikube, to choose a specific minikube driver, please use the `--vm-driver` command line parameter. For example to use the kvm2 driver with clusterctl you would add `--vm-driver kvm2`
```

Comment:
I believe this needs to be passed through using `--bootstrap-flags` instead of passed directly.

Reply:
@detiber it is being fixed at https://github.com/kubernetes-sigs/cluster-api/pull/846